### PR TITLE
UIBULKED-404: Render preview after query executed

### PR DESCRIPTION
**Pull request by UladzislauKutarkin**
https://folio-org.atlassian.net/browse/UIBULKED-404
Here we added `runQueryDataSource` functionality with rendering preview. Also separated functionality for rendering `Preview` for different `criteria`

https://github.com/folio-org/ui-bulk-edit/assets/72550466/ffe6e5b1-7f23-48b2-9b46-7c7d696ee2e9

### CODE REVIEW

**Comment by NikitaSedyx** [comment]
query is not human readable

**Comment by UladzislauKutarkin** [comment]
> query is not human readable

I know about that, we need to discuss it with the backend guys, because  I don't know how we can convert it on the UI side. In `ui-list` backend side returns human readable query

File: package.json [hunk 1]
```diff
@@ -46,7 +46,8 @@
           "bulk-operations.download.item.get",
           "bulk-operations.collection.get",
           "bulk-operations.item.get",
-          "usergroups.collection.get"
+          "usergroups.collection.get",
+          "bulk-operations.item.query.post"
```
Comment (NikitaSedyx) [inline package.json:50]: why not in query permissions?

Comment (UladzislauKutarkin) [reply package.json]: fixed! Thanks for noticing that.

File: src/components/BulkEditList/BulkEditListResult/Preview/Preview.js [hunk 2]
```diff
@@ -6,6 +6,7 @@ import {
 } from '@folio/stripes/components';
 import PropTypes from 'prop-types';
 import { useLocation } from 'react-router-dom';
+import React from 'react';
```
Comment (vashjs) [inline src/components/BulkEditList/BulkEditListResult/Preview/Preview.js:9]: please change order of imports

**Review by vashjs — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -93,6 +93,7 @@
 * [UIBULKED-351](https://issues.folio.org/browse/UIBULKED-351) Hide Query tab
 * [UIBULKED-276](https://issues.folio.org/browse/UIBULKED-276) Rename "Instance" column for holdings records preview
 * [UIBULKED-352](https://issues.folio.org/browse/UIBULKED-352) Localize numbers displayed in bulk edit
+* [UIBULKED-404](https://issues.folio.org/browse/UIBULKED-404)Render preview after query executed.
```
Comment (NikitaSedyx) [inline CHANGELOG.md:96]: space plz

**Review by NikitaSedyx — APPROVED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (UladzislauKutarkin) [reply CHANGELOG.md]: done

File: src/components/BulkEditList/BulkEditListResult/Preview/Preview.js (hunk 2 above)
Comment (UladzislauKutarkin) [reply src/components/BulkEditList/BulkEditListResult/Preview/Preview.js]: done

**Pull request merged**